css/styles.css

```diff
@@ -0,0 +1,358 @@
+@font-face {
+  font-family: cocogoose;
+  src: url("../fonts/cocogoose-pro.ttf");
+}
+@font-face {
+  font-family: cocogoose-it;
+  src: url("../fonts/cocogoose-pro-italic.ttf");
+}
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "lato", serif;
+}
+
+html,
+body {
+  height: 100%;
+}
```

Comment:
- [ ] Please kindly fix your design for mobile and desktop screens, your web should be responsive. Your web has some gaps to the left and right sides, you should verify the container max-width of every screen `media (min-width 576px)`, please kindly fix that 😉 

Your design

<img width="982" alt="Screenshot 2023-04-27 at 01 19 42" src="https://user-images.githubusercontent.com/74501523/234716570-0103c09d-ad0b-4f4b-a795-50fd1d61a418.png">

<img width="564" alt="Screenshot 2023-04-27 at 01 19 20" src="https://user-images.githubusercontent.com/74501523/234716613-9c2189fc-b3c7-4c56-98d7-d3c29e9eef36.png">

Reply:
Thank you. changes done!